media-sound/id3v2/id3v2-0.1.12-r1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="A command line editor for id3v2 tags"
+HOMEPAGE="http://id3v2.sourceforge.net/"
+SRC_URI="mirror://sourceforge/id3v2/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~x86 ~x86-fbsd"
+
+DEPEND="media-libs/id3lib"
```

Comment:
Why do you keep removing `RDEPEND` definitions? This library is clearly being linked to.

Reply:
fixed.
Sorry about that. I've removed it in the first place because i was misunderstanding the devmanual and was taking ```no more RDEPEND="${DEPEND}"``` literally.